### DISCUSSION TITLE
谷歌浏览器，后面会提供其它的下载地址吗？

### DISCUSSION THREAD

**Discussion by Seablip** [chengfengfengwang/lingo-link#14] [General]
谷歌浏览器的版本，因为谷歌应用商店访问不了，后面会提供其它的下载地址吗？云盘之类的下载地址。

**Comment by chengfengfengwang** [comment]
<img width="1255" alt="image" src="https://github.com/chengfengfengwang/lingo-link/assets/23279667/fa60b2c9-a7c8-4778-a8d7-ea4ad88336fa">
release能访问吗？后续会在release同步更新
